src/errors/RequestResponseError.ts

```diff
@@ -64,6 +64,16 @@ const serializeErrorResponse = ({ code, httpCode, errors }: FormattedErrorReques
     return `Code: ${code}, HttpCode: ${httpCode}, Errors: ${formattedErrors.join(", ")}`;
 };
 
+export const fetchErrorReasons = ({ errors, code }: FormattedErrorRequest) => {
+    const formattedErrors = arrify(errors);
+
+    if (formattedErrors.length === 0) {
+        return code;
+    }
+
+    return formattedErrors.map((error) => error.reason);
+};
```

Comment:
Thanks. We have some error parsing function in the node 🙂 We may want to add this extra logic to the error parsing and add the unit tests.

Thanks a lot for adding that! 🎉 It will be super useful on the SDK

Reply:
@phieronymus 
Thanks for checking :bow: 
But, could you tell me more about the following?
> We may want to add this extra logic to the error parsing

(About the test, I'll add it now :+1: )